lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -371,11 +369,17 @@ namespace {
         // case, because of the overload resolution.
         CallArgs.push_back(E);
       }
-      Call = m_Sema->ActOnCallExpr(/*Scope*/0, m_UnresolvedNoAlloc,
+
+      // Test CallArgs.size to make sure an additional argument (the value)
+      // has been pushed on, if not than we didn't know how to handle the type
+      if (CallArgs.size() > nArgs) {
```

Comment:
Could you remove the if stmt? It will still crash without even giving a nice diagnostics before upon unsupported type.